config/kubernetes-sigs/sig-cluster-lifecycle/teams.yaml

```diff
@@ -99,11 +99,15 @@ teams:
     - alexeldeib
     - CecileRobertMichon
     - devigned
+    - dtzar
```

Comment:
@sonasingh46 @marosset should also be added
@alexeldeib @shysank should be removed

Reply:
Done.